net/base/net_export.h

```diff
@@ -4,12 +4,13 @@
 
 #ifndef NET_BASE_NET_EXPORT_H_
 #define NET_BASE_NET_EXPORT_H_
-
+#include "starboard/configuration.h"
```

Comment:
Again, this needs `#ifdef STARBOARD` ( or perhaps COBALT_CUSTOMIZATIONS ) bracket around it

Reply:
done